### PR TITLE
Ch setup dotenv 184638224

### PR DESCRIPTION
**Pull request by AngeNicole**
<b>What does this PR do?</b>

- loads environment variables
- store configurations

<b>Description of Task to be completed</b>

- This is a file that helps in accessing your application's environment variables

<b>How should this be manually tested?</b>

- Cloning this repository to your local machine.
- Switch to a branch named ch-setup-dotenv-184638224
- Open Terminal on your code editor and type yarn
- Create a file in root called .env and insert
image

### CODE REVIEW

**Comment by iradukundairenee** [comment]
@AngeNicole  request at least 3 reviewrs including me ,manager and sprint lead

File: package.json [hunk 1]
```diff
@@ -21,6 +25,7 @@
     "chai-http": "^4.3.0",
     "express": "^4.18.2",
     "mocha": "^10.2.0"
+>>>>>>> c8ffa0ca6b32e4b45bc1473d49b8bc31734e9caa
   },
```
Comment (iradukundairenee) [inline package.json:25]: have you make rebase ?

**Review by iradukundairenee — CHANGES_REQUESTED** [review]
you are not allowed to push .env file

**Comment by iradukundairenee** [comment]
remember to squash your commit 4 commit is not Good practice

**Review by ishgervais — APPROVED** [review]

**Pull request closed without merging**